src/main/java/synapseawsconsolelogin/Auth.java

```diff
@@ -97,9 +92,6 @@
 	static final String AWS_REGION_PARAMETER = "AWS_REGION";
 	static final String SYNAPSE_OAUTH_CLIENT_ID_PARAMETER = "SYNAPSE_OAUTH_CLIENT_ID";
 	static final String SYNAPSE_OAUTH_CLIENT_SECRET_PARAMETER = "SYNAPSE_OAUTH_CLIENT_SECRET";
-	static final String MARKETPLACE_PRODUCT_CODE_SC_PARAMETER = "MARKETPLACE_PRODUCT_CODE_SC";
-	static final String MARKETPLACE_ID_DYNAMO_TABLE_NAME_PARAMETER = "MARKETPLACE_ID_DYNAMO_TABLE_NAME";
```

Comment:
these are params in the AWS parameter.  should we remove these parameters from there as well?

Reply:
Yes, you are right.  We first have to revert changes in other repo's, including synapse-login-aws-infra and organizations-infra.  I will create PRs for those changes.